### PR TITLE
Rename Swagger to OpenAPI Specification

### PR DESCRIPTION
**Pull request by meztez**
… OpenAPI

- NA Update NEWS, no user facing changes I think
- NA Add tests
- [x] Update documentation with `devtools::document()`

### CODE REVIEW

**Comment by schloerke** [comment]
Small comment: Can we move the content of `./R/parsers-body.R` underneath the content of `./R/parse-body.R`? This should all be grouped anyways.

**Comment by meztez** [comment]
> 
> 
> Small comment: Can we move the content of `./R/parsers-body.R` underneath the content of `./R/parse-body.R`? This should all be grouped anyways. (Then delete `./R/parsers-body.R`)

Should we do the same for serializers? There is bunch of them under different files.

**Comment by schloerke** [comment]
@meztez Let's leave them for now.  I'm going to create as `serializer_with_headers(...)` function as a base case for `serializer_content_type()`. This will allow for https://github.com/rstudio/plumber/pull/455 to have a leg to stand on and allow all other serializers to leverage the benefits / remove the boilerplate.

(PR coming in a few days.)

File: R/plumber.R [hunk 1]
```diff
@@ -902,7 +902,13 @@ plumber <- R6Class(
     },
     #' @description Retrieve openAPI file
     openAPIFile = function() {
-      self$swaggerFile()
+      warning("`$openAPIFile()` has been deprecated in v1.0.0 and will be removed in a coming release. Please use `$apiSpec()`.")
```
Comment (cpsievert) [inline R/plumber.R:905]: Consider using lifecycle (if you anticipate wanting to avoid repeated warnings) and/or `.Deprecated`

File: R/plumber.R [hunk 2]
```diff
@@ -902,7 +902,13 @@ plumber <- R6Class(
...
+      self$apiSpec()
+    },
+    #' @description Retrieve openAPI file
+    swaggerFile = function() {
+      warning("`$swaggerFile()` has been deprecated in v1.0.0 and will be removed in a coming release. Please use `$apiSpec()`.")
```
Comment (cpsievert) [inline R/plumber.R:910]: Same as above

File: R/plumber.R (hunk 1 above)
Comment (cpsievert) [reply R/plumber.R:905]: If so, would be good in another PR (this is non-blocking)

**Pull request merged**